error-messages/user.messages.js

```diff
@@ -6,7 +6,8 @@ const USER_NOT_AUTHORIZED = 'USER_NOT_AUTHORIZED';
 const EMAIL_ERROR = 'EMAIL_ERROR';
 const INVALID_PERMISSIONS = 'INVALID_PERMISSIONS';
 const PASSWORD_RECOVERY_ATTEMPTS_LIMIT_EXCEEDED =  'PASSWORD_RECOVERY_ATTEMPTS_LIMIT_EXCEEDED';
-const TOKEN_NOT_VALID = 'TOKEN_NOT_VALID';
+const AUTHENTICATION_TOKEN_NOT_VALID = 'AUTHENTICATION_TOKEN_NOT_VALID';
+const RESET_TOKEN_NOT_VALID = 'RESET_TOKEN_NOT_VALID';
```

Comment:
Please, rename it to `RESET_PASSWORD_TOKEN_NOT_VALID` to clearly show for which recovery this token is used for

Reply:
renamed